chcemvediet/apps/anonymization/templatetags/chcemvediet/anonymization.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from poleno.utils.template import Library
+from chcemvediet.apps.anonymization.anonymization import generate_user_pattern, ANONYMIZATION_STRING
+
+
+register = Library()
+
+@register.simple_tag
```

Comment:
@martinmacko47 Napadlo mi taketo riesenie ako to dat do template. Avsak v kode som to zatial nikde nepouzil

Reply:
:ok: